docs/framework/react/guide/custom-search-param-serialization.md

```diff
@@ -2,50 +2,50 @@
3V3nAzCEAgE4E8L36mBjkJAhCAwAwIzFpwnH/++bu++93vXhrH8ZM8z7tXGIbnzQADt4TAQAJSiVXC9CSk3xgju9dPfWE0z7wjs3hw7tRF8XDwnEOgagRYFKjaiGEvBCDQi8CsBUeXTUuzmMjxZEBgGAERxrOumjkRwTGs43wOAQjMloAsCsRxfNEs4p5n23PuDgEIzBOBkgmOeUJLXyBQKAEER6E4aQwCEIAABCAAgWkRQHBMizT3gcB4BBAc4/HjaghAAAIQgAAEZkQAwTEj8NwWAjkJIDhyAuN0CEAAAhCAAATKQSDPJKYcFmMFBBaTQJ53NXPS+GKipNcQgAAEIAABCEyTAEVqpkmbe0FgdAJ53tXMgkMancQGP6N3kyshAAEIQAACEJg3AnkmMfPWd/oDgSoRSIrVyO7wQ/XE0BNcx2XTP2vtNVTAqdKjgK2LRkDcm7yjizbq9BcC80UAwTFf40lv5pfARASH7DSqlDrIZGZ+Hxx6Vm0C7sX3PO9ivJHVHkush8AiE0BwLPLo0/cqEZiI4CCkqkqPALYuIoE8L/4i8qHPEIBANQggOKoxTlgJgTzzjswhVWCFAAQgAAEIQAACkyaA4Jg0YdqHQDEEEBzFcKQVCEAAAhCAAASmTCDPJGbKpnE7CEAgRSDPu4qHg0cHAhCAAAQgAIFSEZC80SyVb0plNMZAYMEIIDgWbMDpLgQgAAEIQGCeCCA45mk06cs8E8j6ruLhmOengL5BAAIQgAAEKkgg6ySmgl3DZAjMFYGs72omwSG1/YUOJXHn6hmhMxCAAAQgAIFSEpBJDCW+Szk0GAWBHQQKFRyy6Z+0HobhxXCGAATKS4DFgfKODZZBAALZCci8Q2t9kD2FsjPjTAhMm0CeinJ4OKY9OtwPAhMkwOLABOHSNAQgMDUCCI6poeZGEBiZQOGCY2RLuBACEJgqAfmRttZeQ/jjVLFzMwhAoGACeGsLBkpzEJgAAXlPtdYXZomAyuThmICNNAkBCEyAgKw2EIIwAbA0CQEITJVAnonMVA3jZhCAwDaBPAsDCA4eHAhAAAIQgAAESkUgT6hGqQzHGAgsEAEExwINNl2FAAQgAAEIzBuBPBuKzVvf6Q8EqkIgTxh3Zg+HqBjiwqvyCGAnBCAAAQhAoNoEKI1b7fHD+vknIO+oUupgFn2QSXC4lQZqYs//w0MPIQABCEAAAmUgQKWqMowCNkCgN4G8XshMgkNulcdtwuBAAAIQgAAEIACBcQiQxzEOPa6FwGQJJPkblxtjMmmJTCdN1mRahwAEiiCQJ3mriPvRBgQgAIFJEsi7gjpJW2gbAhDYSSCvIwLBwRMEgTkhkPfln5Nu0w0IQGCOCRBWNceDS9cqTSBP/oZ0FMFR6eHGeAj8mIB4ODzPu5p9OHgqIACBeSFAWNW8jCT9mCcCecOpEBzzNPr0BQIQgAAEIDCHBKhWNYeDSpcqTSCvdwPBUenhxngIQAACEIDA/BPAyzH/Y0wPq0PAeTeylsN1PcsdUpUkcV2UpeZudfBhKQQgAAEIQAACZSVALkdZRwa7FomAK+SQV2yM5OFgT45FerToKwQgAAEIQGD2BKhYNfsxwAIIjOrdGElwyEWU3+ShgwAEIAABCEBgmgTEyyH3C8Pw4mnel3tBAALbc//LR/FujCw4AA8BCJSDgIh/rfWF/ACXYzywAgIQmCwBRMdk+dI6BHoRSMruXySfZd3or7ud3DkcDAUEIFAOAuPEUpajB1gBAQhAID8BN/nxPO9iyoDn58cVEMhDIC02xnnnEBx5qHMuBEpGQEQHP7glGxTMgQAEJk4gKcsp9zlIEZuJ4+YGC0jAVYez1nY8G+OIDbkewbGADxFdhgAEIAABCFSZgKuYqZSSmHKER5UHE9tLRcAJDWeUCI5xxUZhgoPQjlI9KxgDAQhAAAIQWAgCqao5rr94PBZi5Olk0QTSQsNae43kh3aEgtYHi4ikKMzDMco250XDoj0IzCsBQqfmdWTpFwQgUASBbuGhtb5aJk2e511dxGSpCBtpAwJlIiDzCrHHWnu5eDHcO1O00HB9LkxwlAkitkBgngiMU/d6njjQFwhAAALDCLiy/alQq84lbjIlfy9ahLiJ2zDbBn0ex3Fn8lfE4SaMRbSVpw0X65/nmmmfK8/BJO8pInec9uXZzHJ9HhGdfj7lOXPPR3q8kvejIzqK8mh09wPBkWVkOQcCMyTgSt9O6ktghl3j1hCAAAQmSsAJkPQkvAoT4zSUvJPkcSe9TpTlHZg8k+C8bRd1/iBxWIToGyT2yvbcuedK5hbCd9LjNxXB4VZopXPsF1DUa0M780Iglfwo9a2vmJd+0Q8IQAACVSBQhIdC+lnEhLUXr1l4LMo2OZ7Gc5RX2OW1aRwhmNXzkbZp0gIib/+nIjjEKBEd/dyYLuk87cpJJ6/0W9l1K78yiP0makn94L6fZ5nsuXP63cN9PshN677QBj0Aw+L0s7SR9wGY9/PHZZplbIc9Y8Oe01SNa5Id5/2BpH8QgAAEIACBBSQwNcExjK1ze6Yn9cNCSYZN5JzQkdWBQaJFYj37eV9SFbikuZ4TQjdhzNJGv9Ji6YS3Xrs4pu3o18YwOxIeVv4c1sag2uau/vmwvgzyaI3bRhE88rTRj0daLPfz3mV5Toe9H3wOAQhAAAIQgAAEqkqgNIKjqgCz2p3FOzHsnCyfi0t3kKell7BL98F9Pk4bWbwCw+zI2saw5L8szKT/ZXM9Zn2uOA8CEIAABCAAAQiUnQCCo+wjhH0QgAAEIAABCEAAAhCoMAEER4UHD9MhAAEIQAACEIAABCBQdgIIjrKPEPZBAAIQgAAEIAABCECgwgQQHBUePEyHAAQgAAEIQAACEIBA2QkgOMo+QtgHAQhAAAIQgAAEIACBChNAcFR48DAdAhCAAAQgAAEIQAACZSeA4Cj7CGEfBCAAAQhAAAIQgAAEKkzg/wGL5FYq12posgAAAABJRU5ErkJggg==">
```

Comment:
I've asked a quesiton in the website maintainers channel about how these kinds of images should be handled.

Reply:
I've removed it for now. Can add it later.